.changesets/fix_renee_operation_variables.md

```diff
@@ -0,0 +1,11 @@
+### Fix GraphQL query directives validation bug
```

Comment:
I'm not sure if it's worth adding a changeset for this, because as far as we know, none of our users have any query that does this...

Reply:
I'd keep the changeset. It's still a fix, even if it doesn't actually affect anyone.